source/hardware/controllers/dj_techtools_midi_fighter_spectra.rst

```diff
@@ -0,0 +1,128 @@
+DJ TechTools MIDI Fighter Spectra
+=================================
+
+`Manufacturer's product page <https://store.djtechtools.com/products/midi-fighter-spectra>`_ ·
+`Forum thread <https://mixxx.discourse.group/t/dj-techtools-midi-fighter-spectra/31554>`_
+
+.. versionadded:: 2.5.1
+
+Overview
+--------
+
+The MIDI Fighter Spectra contains 16 buttons with programmable LEDs, white
+ground effect lighting, and six buttons on the side.
+
+.. figure:: ../../_static/controllers/dj_techtools_midi_fighter_spectra.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: MIDI Fighter Spectra with left buttons labeled L1–L3, right buttons labeled R1–R3, and arcade buttons labeled from 1 in the top left to 16 in the bottom right.
+   :figclass: pretty-figures
+
+Out of the box all six side buttons are mapped to the layer selection functions
+and cannot be used for other features.
+
+.. csv-table::
+   :header: "#", "Function"
+   :widths: 15 70
+
+   "L1", "Previous Layer"
+   "L2", "Previous Layer"
+   "L3", "Previous Layer"
+   "R1", "Next Layer"
+   "R2", "Next Layer"
+   "R3", "Next Layer"
+
+Layer 1
+-------
+
+The first layer of buttons control the EQ and quick effect cutoffs.
+
+.. csv-table::
+   :header: "#", "Deck", "Function"
+   :widths: 15, 15, 70
+
+   "1", "3", "Kill High"
+   "2", "1", "Kill High"
+   "3", "2", "Kill High"
+   "4", "4", "Kill High"
+   "5", "3", "Kill Mid"
+   "6", "1", "Kill Mid"
+   "7", "2", "Kill Mid"
+   "8", "4", "Kill Mid"
+   "9", "3", "Kill Low"
+   "10", "1", "Kill Low"
+   "11", "2", "Kill Low"
+   "12", "4", "Kill Low"
+   "13", "3", "Toggle Quick Effect"
+   "14", "1", "Toggle Quick Effect"
+   "15", "2", "Toggle Quick Effect"
+   "16", "4", "Toggle Quick Effect"
```

Comment:
All rows have the same action mapped, I think we can simplifiy this large table
For example
* show the deck -> columns mapping (3 1 2 4) in the header or in the layer description
* combine pad numbers per row (1-4, 2-8 etc.

Then we'd have only five rows.

Reply:
Done!